cmd/medius/common/registry.go

```diff
@@ -100,7 +94,7 @@ func gatherArtifacts(registry *[]Entry, gatherers []api.ArtifactsGatherer) {
 		} else {
 			for i := range artifacts {
 				*registry = append(*registry, Entry{
-					Artifact:     artifacts[i],
+					Artifacts:    []api.Artifact{artifacts[i]},
```

Comment:
Can you please extend the GathererInterface to return `[][]api.Artifact`?

Reply:
Why `[][]api.Artifact`? Do you mean to array of array?

Reply:
Exactly, so it fits the use case of providing multiple architectures per artifact.

Reply:
Do you mean to `artifacts["arm64"][i]`?